indico/modules/events/templates/display/cern_committees.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'events/display/administrative/event.html' %}
+
+{% block header %}
+    <table class="eventHeader">
+        <tr>
+            <td>
+                <div style="float:left;">
+                    <img src="{{ indico_config.config.ImagesBaseURL }}/images/smallcern.png">
```

Comment:
This is very strange... `indico_config.config` is the legacy Config instance

```

In [6]: config.imagesBaseURL
Out[6]: '/images'

In [7]: config.config.ImagesBaseURL
AttributeError...
```

(`config` in the shell is the same as `indico_config` in jinja templates)

I believe it only works for you since Jinja converts the AttributeError to `Undefined` which shows up as an empty string.